NostalgiaForInfinityNext.py

```diff
@@ -2423,14 +2423,41 @@ def load_hold_trades_config(self):
                 log.error("Failed to load JSON from %s: %s", hold_trades_config_file, exc)
             else:
                 trade_ids = hold_trades_config.get("trade_ids")
+                trade_pairs = hold_trades_config.get("trade_pairs")
 
             if not trade_ids:
                 return
 
             open_trades = {
                 trade.id: trade for trade in Trade.get_trades_proxy(is_open=True)
             }
-
+            if isinstance(trade_pairs, dict):
+                # New syntax
+                for trade_pair, profit_ratio in trade_pairs.items():
+                    try:
+                        trade_pair = str(trade_pair)
+                    except TypeError:
+                        log.error(
+                            "The trade_id(%s) defined under 'trade_pairs' in %s is not a string",
```

Comment:
```suggestion
                            "The trade_pair(%s) defined under 'trade_pairs' in %s is not a string",
```
Based on the above comment, the error message will have to be different.